### PR TITLE
Feat/containers

### PR DESCRIPTION
**Pull request by sjpb**
Add `podman`, `podman-compose` and `apptainer` to image.

### CODE REVIEW

**Comment by sjpb** [comment]
Will be image `openhpc-230214-1026.qcow2`

**Comment by sjpb** [comment]
Will be image `openhpc-230214-1147.qcow2`

**Comment by sjpb** [comment]
fff34b1 reduced image size from 8.01GB to 7.91GB.

**Comment by sjpb** [comment]
Built image `openhpc-230215-1202.qcow2`

File: roles/builder/defaults/main/users.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+prometheus_db_dir: /var/lib/prometheus # will get moved to appliances state dir by appliance
```
Comment (m-bull) [inline roles/builder/defaults/main/users.yml:2]: Should this go in a `prometheus.yml` file?

File: roles/builder/tasks/users.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# From ansible/bootstrap.yml
+
+    - name: Add groups
```
Comment (m-bull) [inline roles/builder/tasks/users.yml:3]: Remove whitespace from the start of all lines in here

**Review by m-bull — CHANGES_REQUESTED** [review]

**Comment by sjpb** [comment]
Built image `openhpc-230217-1440.qcow2`

**Review by m-bull — APPROVED** [review]

**Pull request merged**